Gems/OpenXRVk/Code/Source/OpenXRVkSwapChain.cpp

```diff
@@ -241,7 +241,7 @@ namespace OpenXRVk
     VkFormat SwapChain::SelectColorSwapChainFormat(const AZStd::vector<int64_t>& runtimeFormats) const
     {
         // List of supported color swapchain formats.
-        constexpr int64_t SupportedColorSwapchainFormats[] = { VK_FORMAT_B8G8R8A8_UNORM, VK_FORMAT_R8G8B8A8_UNORM };
+        constexpr int64_t SupportedColorSwapchainFormats[] = { VK_FORMAT_B8G8R8A8_UNORM, VK_FORMAT_R8G8B8A8_UNORM, VK_FORMAT_R8G8B8A8_SRGB };
```

Comment:
On the VR, I also encountered this problem, you can't completely modify this problem, in the VR picture is abnormal.

Reply:
Yeah, I think I need to find a way so that the rest of the rendering pipelines render targets also use sRGB color-space if the device requires sRGB. Otherwise, we could explicitly do gamma correction in the shader code. Would love @o3de/sig-graphics-audio feedback. At the very least, this PR allows for testing on device, which is better than nothing :D

Reply:
Got it